app/js/elements/fxos-customizer.js

```diff
@@ -1,12 +1,12 @@
 (function(window) {
+/*jshint maxlen:false*/
```

Comment:
Ah, I forgot to remove this before pushing to the PR. I previously had the font icon embedded base64 code in here which is why I needed this. Its no longer loaded here though, so this can be removed.